packages/utils/src/utils.ts

```diff
@@ -84,6 +84,11 @@ export function ensureBuffer(input: string|Buffer, encoding: BufferEncoding = 'u
     throw new Error(`Invalid input given, expected string or buffer, got ${getTypeOf(input)}`);
 }
 
+export function isBoolean(input: any): boolean {
```

Comment:
It is important to use the is keyword so the type for the input will be updated outside of this function.

```suggestion
export function isBoolean(input: any): input is boolean {
```